README.md

```diff
@@ -0,0 +1,52 @@
+
+# Hello Microverse
+
+This is project is going to help us to master github flow and get used with linters for checking errors in our css ans Html files.
+
+## Materials required for validation
+
+- npm install --save-dev hint@6.x
+- npx hint .
+- npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x       stylelint-csstree-validator@1.x
+
```

Comment:
- [ ] **OPTIONAL**
Please make sure that there are non-technicals (people who are not related to programming) who might view your projects in the future. So, based on this, your README file should be human-readable.
I can see that you're trying to tell how to set up the Linters correctly which is a great thing you did (Keep it up 👍) but don't you think it'd be better and cleaner if you just included the link of the [Linter's repo](https://github.com/microverseinc/linters-config/tree/master/html-css) to do so! Since it's an additional thing to know and it's not needed to set up the project locally!